### ISSUE TITLE
React: Card section - simple - beginning and ending boundary for the card is not conveyed for the screen reader user. 

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4613]
## Detailed description

Issue: 
1. The beginning and ending boundary for the card is not conveyed for the screen reader user. 
2. In tab / shift tab navigation, when focus reaches the link in each card, Screen reader announces a lengthy text which makes it oververbose.

Expected:
1. Provide each card within the landmark region and give appropriate label.
2. It may be possible if the entire content in the card is given in anchor tag. In this scenario, only the heading can be given in anchor tag. Note: the link text/label should be concise and convey only the relevant information. 

Note: working fine on react version.

## Additional information
- this fix should address the bug included in issue (#5504 ). Engineer should check to be sure this fix addresses the bug included in (#5504 )
 - Reach out to kenny for additional details.

**Comment by kennylam** [comment]
@Praveen-sr I'm not sure this is the correct approach for card links. As I understand it, `landmark` roles are meant to identify larger regions of a page, not individual elements within it. I also don't understand why we'd want to reduce the amount of copy read by screen readers, since it is meant to inform users of the content they'd expect to see when clicking on it.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role#Accessibility_concerns

cc: @jeffchew @RobertaJHahn

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested the PR provided, here is the report 

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/51fb7e45-af3a-4b94-bf7b-305df78121dd)

**Comment by RobertaJHahn** [comment]
@ariellalgilmore Can this issue and (#5504 ) be closed?

**Comment by ariellalgilmore** [comment]
@RobertaJHahn We just confirmed with the team the changes that will be made for this issue to be closed, so next sprint I will finish this issue. 5504 and mostly all the other card accessibility bugs require this fix being made.

**Comment by ariellalgilmore** [comment]
@RobertaJHahn only the React version has been completed

**Comment by RobertaJHahn** [comment]
@ariellalgilmore Thanks for reminding me.  I have created issue (#6005 ) and assigned it to you in the next sprint.  Closing this issue.
